### PR TITLE
cutdown flake8,  added autopep8

### PR DESCRIPTION
**Pull request by nabobalis**
Removes flake8 which just tells you your code is bad with a program that fixes it for you.

autopep8 fixes:

```
E101 - Reindent all lines.
E11  - Fix indentation.
E121 - Fix indentation to be a multiple of four.
E122 - Add absent indentation for hanging indentation.
E123 - Align closing bracket to match opening bracket.
E124 - Align closing bracket to match visual indentation.
E125 - Indent to distinguish line from next logical line.
E126 - Fix over-indented hanging indentation.
E127 - Fix visual indentation.
E128 - Fix visual indentation.
E129 - Fix visual indentation.
E131 - Fix hanging indent for unaligned continuation line.
E133 - Fix missing indentation for closing bracket.
E20  - Remove extraneous whitespace.
E211 - Remove extraneous whitespace.
E22  - Fix extraneous whitespace around keywords.
E224 - Remove extraneous whitespace around operator.
E225 - Fix missing whitespace around operator.
E226 - Fix missing whitespace around arithmetic operator.
E227 - Fix missing whitespace around bitwise/shift operator.
E228 - Fix missing whitespace around modulo operator.
E231 - Add missing whitespace.
E241 - Fix extraneous whitespace around keywords.
E242 - Remove extraneous whitespace around operator.
E251 - Remove whitespace around parameter '=' sign.
E252 - Missing whitespace around parameter equals.
E26  - Fix spacing after comment hash for inline comments.
E265 - Fix spacing after comment hash for block comments.
E266 - Fix too many leading '#' for block comments.
E27  - Fix extraneous whitespace around keywords.
E301 - Add missing blank line.
E302 - Add missing 2 blank lines.
E303 - Remove extra blank lines.
E304 - Remove blank line following function decorator.
E305 - Expected 2 blank lines after end of function or class.
E306 - Expected 1 blank line before a nested definition.
E401 - Put imports on separate lines.
E402 - Fix module level import not at top of file
E501 - Try to make lines fit within --max-line-length characters.
E502 - Remove extraneous escape of newline.
E701 - Put colon-separated compound statement on separate lines.
E70  - Put semicolon-separated compound statement on separate lines.
E711 - Fix comparison with None.
E712 - Fix comparison with boolean.
E713 - Use 'not in' for test for membership.
E714 - Use 'is not' test for object identity.
E721 - Use "isinstance()" instead of comparing types directly.
E722 - Fix bare except.
E731 - Use a def when use do not assign a lambda expression.
W291 - Remove trailing whitespace.
W292 - Add a single newline at the end of the file.
W293 - Remove trailing whitespace on blank line.
W391 - Remove trailing blank lines.
W503 - Fix line break before binary operator.
W504 - Fix line break after binary operator.
W601 - Use "in" rather than "has_key()".
W602 - Fix deprecated form of raising exception.
W603 - Use "!=" instead of "<>"
W604 - Use "repr()" instead of backticks.
W605 - Fix invalid escape sequence 'x'.
W690 - Fix various deprecated code (via lib2to3).
```

### CODE REVIEW

File: sunpy/coordinates/tests/test_wcs_utils.py [hunk 1]
```diff
@@ -312,7 +312,8 @@ def test_frame_mapping_obsgeo_spherical(dkist_location, caplog):
     loc_sph = location.spherical
     wcs = WCS(naxis=2)
     wcs.wcs.ctype = ['HPLT', 'HPLN']
-    wcs.wcs.obsgeo = [0, 0, 0] + [loc_sph.lon.to_value(u.deg), loc_sph.lat.to_value(u.deg), loc_sph.distance.to_value(u.m)]
+    wcs.wcs.obsgeo = [0, 0, 0] + \
+        [loc_sph.lon.to_value(u.deg), loc_sph.lat.to_value(u.deg), loc_sph.distance.to_value(u.m)]
```
Comment (Cadair) [inline sunpy/coordinates/tests/test_wcs_utils.py:316]: Can we turn off auto line wrapping outside of parenthesis? Line continuation characters are evil.

**Review by Cadair — COMMENTED** [review]
Seems like a generally good idea.

File: sunpy/coordinates/tests/test_wcs_utils.py (hunk 1 above)
Comment (nabobalis) [reply sunpy/coordinates/tests/test_wcs_utils.py:316]: I will work that out.

**Review by Cadair — APPROVED** [review]

**Comment by wtbarnes** [comment]
A diff of what checks are supported/not supported with autopep8 vs flake8 would be good to see before we merge this.

**Comment by Cadair** [comment]
E22, E224 are probably the ones I think are the most important but none of them I consider crtical.

**Comment by nabobalis** [comment]
> E22, E224 are probably the ones I think are the most important but none of them I consider crtical.

I just double checked, it does actually check those.

**Comment by nabobalis** [comment]
Now we have lost nothing.

**Review by wtbarnes — APPROVED** [review]

**Review by Cadair — APPROVED** [review]
Nabil adding blank lines, hell hath frozen over.

**Pull request merged**